Docs: Installation refactor Linux (Split PR #5830) (#7671)
arangodb/arangodb: Documentation/Books/Manual/Installation/Linux.md

Installing ArangoDB on Linux
============================

To install ArangoDB on Linux:

1. Visit the official [Download](https://www.arangodb.com/download) page of the
   ArangoDB web site and download the correct package for your Linux distribution.
   You can find binary packages for the most common distributions there. Linux Mint: 
   please use the corresponding Ubuntu or Debian packages. 

> 1. Follow the installation instructions included in the _Download_ page to use your

Author: ```suggestion
2. Follow the installation instructions on the _Download_ page to use your
```

   favorite package manager for the major distributions. After setting up the ArangoDB
   repository you can easily install ArangoDB using _yum_, _aptitude_, _urpmi_ or _zypper_.

{% hint 'info' %}

> Starting from version 3.4.0 in addition to installation packages (distribution dependent)

Author: ```suggestion
In addition to installation packages (distribution dependent) a `tar.gz` archive
is available starting from version 3.4.0.
```
Does this archive actually contain a universal binary that was talked about some time ago? If so, should we mention it?

Reviewer: @Simran-B I believe so. We should check with engineering and in case improve later the description we added imho

> a _tar.gz_ archive is available.

Author: ```suggestion
```

{% endhint %}

After installation, you may start ArangoDB in several ways. The exact start-up command
depends on your Linux distribution, as well as on the type of ArangoDB deployment you

> are interested on (_Single Server_, _Master-Slave_, _Active Failover_, _Cluster_, _DC2DC_).

Author: ```suggestion
are interested in (_Single Server_, _Master-Slave_, _Active Failover_, _Cluster_, _DC2DC_).
```

> Please refer to the [_Deployment_](../Deployment/README.md) chapter for further information

Author: ```suggestion
Please refer to the [_Deployment_](../Deployment/README.md) chapter for details.
```

> on deployment.

Author: ```suggestion
```

Securing your Installation
--------------------------

### Debian / Ubuntu

Debian based packages will ask for a password during installation. 

#### Securing Unattended Installations on Debian

For unattended installations, you can set the password using the
[debconf helpers](http://www.microhowto.info/howto/perform_an_unattended_installation_of_a_debian_package.html):

```
echo arangodb3 arangodb3/password password NEWPASSWORD | debconf-set-selections
echo arangodb3 arangodb3/password_again password NEWPASSWORD | debconf-set-selections
```

The commands above should be executed prior to the installation.

### Red-Hat / CentOS

Red-Hat based packages will set a random password during installation. The generated
random password is printed during the installation. Please write it down somewhere,
or change it to a password of your choice by executing:

```
ARANGODB_DEFAULT_ROOT_PASSWORD=NEWPASSWORD arango-secure-installation
```

The command should be executed after the installation.

### Other Distributions

> For other distributions to set a _root_ password, run `arango-secure-installation`.

Author: ```suggestion
For other distributions run `arango-secure-installation` to set a _root_ password.
```
Should we tell users not to run this if they already have data and users configured?

Reviewer: I thought the same. I think we still have to add the explanations on arango-secure-installation, right? I will add a sentence, something like alert: "Please be aware that running `arango-secure-installation` on a server will remove all current database users but root".
